pages/billing/subscription.en-US.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Subscription
+ogImageTitle: Subscription
+ogImageSubtitle: Subscription on Zeabur
+---
+
+# Subscription
+
+## What is Subscription?
+
+There're 3 types of plans on Zeabur: **Free Plan**, **Developer Plan** and **Team Plan**.
+You can check the details of each plan on [plans](/billing/plans) page.
+
+To use Zeabur, you need to subscribe to one of the plans, the default plan is **Free Plan**.
+You can deploy your application with **Free Plan**, but there will be some limitations.
+
+## How to Subscribe?
+
+Zeabur's subscription is based on [Stripe](https://stripe.com/).
+
+There're 2 ways to subscribe to a plan: **Credit Card** and **Alipay**.
+
+### Subscribe with Credit Card
+
+You can subscribe to a plan with your credit card, paying with USD.
+
+- Setup without subscription: Go to [billing page](https://dash.zeabur.com/account/billing), you can setup your credit card in **Payment Method** section. After you've input your credit card information, click **Add** button to save your credit card, then you can simply click **Subscribe** button in [plan page](https://dash.zeabur.com/update-plan) to update your subscription.
+- Setup with subscription: Go to the [plan page](https://dash.zeabur.com/update-plan) in dashboard, choose a plan, and then input your credit card information, click **Subscribe** button, you'll be redirected to Stripe's checkout page, fill the form and you'll be subscribed to the plan with your credit card saved.
+
+### Subscribe with Alipay
+
+If you don't have a credit card that supports Stripe service, you can subscribe to a plan with your Alipay account, paying with CNY.
```

Comment:
How about: *If you don't have a credit card that is compatible with Stripe service*